demos/common/cpp/models/src/model_base.cpp

```diff
@@ -34,9 +34,6 @@ std::shared_ptr<ov::Model> ModelBase::prepareModel(ov::Core& core) {
     // -------------------------- Reading all outputs names and customizing I/O tensors (in inherited classes)
     prepareInputsOutputs(model);
 
-    /** Set batch size to 1 **/
-    ov::set_batch(model, 1);
```

Comment:
@Wovchena Since `MO` can return IR with batch, should we move to separate `setBatch` method?

Reply:
Alternative solution could be to modify `prepareInputsOutputs()` for every model wrapper. I don't see any reason to prefer one or another. Let's proceed with your suggestion to introduce `setBatch` method.

Reply:
Done